src/app/register/page.tsx

```diff
@@ -2,95 +2,105 @@ import { Input } from "@/components/ui/input";
 import { Label } from "@/components/ui/label";
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
 import { registerAgent } from "../actions";
-import { SubmitButton } from "./submit-button";
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import { SubmitButton } from "@/components/SubmitButton";
 
 export default function RegistrationForm() {
   return (
-    <Card className="w-full max-w-2xl mx-auto">
-      <CardHeader>
-        <CardTitle>Agent Registration</CardTitle>
-      </CardHeader>
-      <CardContent>
-        <form className="space-y-6" action={registerAgent}>
-          <div className="space-y-4">
-            <h3 className="text-lg font-semibold">Developer Information</h3>
-            <div className="grid grid-cols-2 gap-4">
-              <div>
-                <Label htmlFor="developer.name">Name *</Label>
-                <Input id="developer.name" name="developer.name" required />
+    <div className="flex flex-col items-center justify-center h-screen -mt-10">
```

Comment:
Not sure we need that negative margin. In general these lead to ticking time bomps. Why is it needed?